lib/oli_web/user_auth.ex

```diff
@@ -97,8 +96,7 @@ defmodule OliWeb.UserAuth do
         "browser_timezone",
         "author_token",
         "author_live_socket_id",
-        "current_author_id",
-        "datashop_session_id"
```

Comment:
Here we do not want to retain the datashop_session_id when a session is ended. Otherwise, we would simply be tracking a user forever and not just their session